docs/push.md

```diff
@@ -8,9 +11,12 @@
 
   **Examples**
    
-  \- Uploads Data Package to DataHub
+  \- Uploads Data Package to DataHub in current working directory
 
     ■ data push
+    
+  \- Uploads Data Package to DataHub with path:
 
-  Current working directory should have datapackage.json
-
+    ■ data push core/finance-vix/
+  
+  Working directory should have datapackage.json
```

Comment:
This is incorrect now. Because not working directory, but `core/finance-vix/` must have datapackage.json